internal/pkg/template/templates/workloads/services/rd-web/cf.yml

```
@@ -82,6 +82,20 @@ Resources:
           ImageRepositoryType: !Ref ImageRepositoryType
           ImageConfiguration:
             Port: !Ref ContainerPort
+            {{- if hasRuntimeEnvironmentSecrets .}}
+            RuntimeEnvironmentSecrets:
+            {{- range $name, $secret := .RuntimeEnvironmentSecrets}}
+            - Name: {{$name}}
+              Value: {{if not $secret.RequiresSub }}{{$secret.ValueFrom}}{{- else}} !Sub 'arn:${AWS::Partition}:{{$secret.Service}}:${AWS::Region}:${AWS::AccountId}:{{$secret.ValueFrom}}' {{- end }}
```

indentation needs to be consistent with variables section below and we don't need `-` because it doesn't create extra spaces as they are in the same line.
```suggestion
              - Name: {{$name}}
                Value: {{if not $secret.RequiresSub }}{{$secret.ValueFrom}}{{else}} !Sub 'arn:${AWS::Partition}:{{$secret.Service}}:${AWS::Region}:${AWS::AccountId}:{{$secret.ValueFrom}}' {{end }}
```

---

Done.